examples/cache/tests/cache_test.bal

```diff
@@ -11,12 +11,12 @@ public function mockPrint(any|error... val) {
     outputs.push(val.reduce(function (any|error a, any|error b) returns string => a.toString() + b.toString(), "").toString());
 }
 
-@test:Config {}
+//@test:Config
```

Comment:
Is there any reason to unmark these functions as testerina functions?

Reply:
I had disabled the test temporarily due to some issues in the local machine. those are re-enabled now.